moped-editor/src/views/projects/projectView/ProjectComponents/ProjectName.js

```diff
@@ -1,21 +1,38 @@
-import makeStyles from '@mui/styles/makeStyles';
+import makeStyles from "@mui/styles/makeStyles";
 import { Typography } from "@mui/material";
 
 const useStyles = makeStyles(() => ({
   titleDisplayField: {
     display: "flex",
   },
+  titleText: {
+    // textOverflow: "ellipsis",
```

Comment:
if you want to test the ellipsis, uncomment this and change over to `overflow: "hidden"`